user_guide/configuration.rst

```diff
@@ -35,13 +39,20 @@ executing your feature suite.
 
 The default profile is always ``default``. All other profiles inherit
 parameters from the ``default`` profile. If you only need one profile, define
-all of your parameters under the ``default:`` root:
+all of your parameters under the ``default`` profile:
 
-.. code-block:: yaml
+.. code-block:: php
 
-    # behat.yml
-    default:
-        #...
+    <?php
+    // behat.php
+    use Behat\Config\Config;
+    use Behat\Config\Profile;
+
+    return new Config()
+        ->withProfile(
+            new Profile('default')
+        )
```

Comment:
In the previous example there was a line with `#...` indicating where you would add additional configuration, maybe we could do the same here with a `//...` line after the `new Profile` line

Reply:
done :)